docs/mp/persistence.adoc

```diff
@@ -0,0 +1,1430 @@
+managed]
+
+* Ensure the following `<dependency>` elements are present as child
+  elements of your project's `pom.xml` file's `<dependencies>` element:
++
+[source,xml]
+----
+<dependency>
+    <groupId>jakarta.transaction</groupId>
+    <artifactId>jakarta.transaction-api</artifactId>
+    <scope>provided</scope> <!--1-->
+</dependency>
+<dependency>
+    <groupId>io.helidon.integrations.cdi</groupId>
+    <artifactId>helidon-integrations-cdi-jta-weld</artifactId>
+    <scope>runtime</scope> <!--2-->
+</dependency>
+----
+<1> The `scope` is `provided`, which ensures that the
+    https://jakarta.ee/specifications/transactions/2.0/apidocs/jakarta/transaction/transactional[JTA
+    classes required for compilation] are available at compile time.
+<2> The implementation of these API classes (provided by
+    https://narayana.io/[Narayana]) will be available at runtime.
+
+=== Configuration [[JTA-Configuration]]
+
+==== Overview
+
+Helidon MP's Jakarta Transactions integration does not require
+configuration, but configuration is possible.  Because configuration
+is of the https://narayana.io/[underlying Narayana transaction
+engine], any restrictions are those of the engine, not of Helidon
+itself.
+
+Narayana, unlike Helidon MP, does not use MicroProfile Config, so its
+configuration options are less flexible.
+
+Some common examples of Narayana configuration follow.
+
+==== Configuring the Object Store Directory
+
+Narayana features an object store directory which it uses to store
+information about transaction outcomes.  To set its location, you may
+set the
+https://www.narayana.io/docs/api/com/arjuna/ats/arjuna/common/ObjectStoreEnvironmentBean.html#setObjectStoreType-java.lang.String-[`ObjectStoreEnvironmentBean.objectStoreDir`]
+system property to the full path of a writeable directory:
+
+[source,shell]
+----
+java -DObjectStoreEnvironmentBean.objectStoreDir=/var/tmp # ...
+----
+
+See
+https://www.narayana.io//docs/project/index.html#d0e4013[Specifying
+the object store location] for more information.
+
+==== Configuring the Default Transaction Manager Timeout
+
+To configure Narayana's
+https://www.narayana.io/docs/api/com/arjuna/ats/arjuna/common/CoordinatorEnvironmentBean.html#setDefaultTimeout-int-[default
+transaction manager timeout], set the
+`com.arjuna.ats.arjuna.coordinator.defaultTimeout` system property to
+an integral value in seconds:
+
+[source,shell]
+----
+java -Dcom.arjuna.ats.arjuna.coordinator.defaultTimeout=60 # ...
+----
+
+For more on configuring Narayana, see
+https://www.narayana.io//docs/project/index.html#chap-JBossJTA_Installation_Guide-Test_Chapter[Setting
+Properties] in the Naryana documentation.
+
+=== Usage [[JTA-Usage]]
+
+To use Helidon MP's Jakarta Transactions integration, annotate a
+method with the https://jakarta.ee/specifications/transactions/2.0/apidocs/jakarta/transaction/transactional[`jakarta.transaction.Transactional`] annotation:
```

Comment:
All specifications (jakarta or mp) related links must use a base-url attribute define in attributes/includes.adoc